TypeCobol/Compiler/Nodes/Data.cs

```diff
@@ -16,38 +16,72 @@ namespace TypeCobol.Compiler.Nodes {
     public class DataDivision: GenericNode<DataDivisionHeader>, Parent<DataSection> {
 
         public const string NODE_ID = "data-division";
+
+        /// <summary>
+        /// Lists indices for data sections
+        /// </summary>
+        private enum SectionIndex
+        {
+            FileSection,
```

Comment:
As this  enum is used to get named integer constants, could you add the first number here `FileSection = 0,`. This is equivalent to what you've written but this emphasizes the fact that the numeric value of each member is of importance.

Reply:
Done